foreign_cc/private/framework.bzl

```diff
@@ -458,16 +458,19 @@ def cc_external_rule_impl(ctx, attrs):
     # TODO: `additional_tools` is deprecated, remove.
     legacy_tools = ctx.files.additional_tools + ctx.files.tools_deps
 
+    inputs_from_tools, manifests_from_tools = ctx.resolve_tools(tools = attrs.tools_deps)
```

Comment:
Possibly should be `legacy_tools` here?

Reply:
Also need to be aware of what this looks like once deprecated attributes are removed. build_data is not included in legacy_tools at the moment.